client/src/components/layout/MainLayout.js

```diff
@@ -63,152 +63,134 @@ const Button = styled.button`
     background: #0056b3;
   }
```

Comment:
> The `MainLayout.js` is a very big file containing many reusable components, such as `Button`, `Input`, etc. For better modularity, consider separating these elements into individual components and importing them instead.

I will not separate MainLayout.js. It's a good advice, but I have to consider my partner's situation. If I did, it means it will add a level in structure, which is not good for a new learner. Just like read a 200 lines text is easier than read 10 files of 20lines text, because 10 files refer each other. I'm making balance between my partner and good structure.